explorer/graphql/resolvers/messages_test.go

```diff
@@ -90,7 +90,7 @@ func TestXMsgsNoCursor(t *testing.T) {
 							Cursor
 							Node {
 								ID
-								Offset
+								MsgOffset
```

Comment:
The prefix doesn't need to be added to the front-end API because the nodes here are of type `XMsg`. The second problem is that it actually introduces a breaking changes for the FE and Explorer. You can leave it and I will rebase and change it in my upcoming PR

Reply:
sorry typo